apm-lambda-extension/cli/build-and-publish.js

```diff
@@ -38,7 +38,7 @@ function getLastJsonFromShellOutput (output) {
 function buildAndPublish () {
   return new Promise(function (resolve, reject) {
     if (!process.env.ELASTIC_LAYER_NAME) {
-      process.env.ELASTIC_LAYER_NAME = 'apm-lambda-extension'
+      process.env.ELASTIC_LAYER_NAME = 'elastic-apm-lambda-extension'
```

Comment:
Maybe removing `lambda` from the name as it's implied from the context and also included in the arn name. See also https://github.com/elastic/apm-aws-lambda/issues/110#issuecomment-1027396444.
```suggestion
      process.env.ELASTIC_LAYER_NAME = 'elastic-apm-extension'
```